### ISSUE TITLE
Completed/Started Date in Anime/Manga List Columns

### ISSUE THREAD

**Issue by Rapptz** [pilar6195/AniList-Extras#19]
I'm not sure how difficult it would be to do this (especially without making a secondary API call...) but it'd be nice to have extra columns in the anime/manga list for the completed/started dates similar to how you can configure it in MAL.

**Comment by pilar6195** [comment]
I think I have an idea what you mean but if you could give me an example of what you're asking for that'll be better.

Also, a bit busy with some other projects. If I decided to implement this, it'll probably be a little while before you see it added.

**Comment by Rapptz** [comment]
Sure, like in my old [MAL List](https://myanimelist.net/animelist/rapptz) there's columns for started and finished dates. 

I wouldn't mind implementing it myself, though the biggest hang-up I had was how to do it without doing another API call but I don't think it's possible to do it that way due to userscript limitations.

**Comment by pilar6195** [comment]
Yeah that's pretty much what I had in mind. I got a few fixes in the backlog to push first but I'll probably have a bit of free time tomorrow to take a look into implementing it. A API request is more than likely going to be needed but it's not really that big of a deal.
